addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -92,6 +100,14 @@ func _can_drop_data(at_position: Vector2, data: Variant) -> bool:
 
 
 func _drop_data(at_position: Vector2, data: Variant) -> void:
+	var type = data.get("type", "")
+	if type == "nodes":
+		_drop_node(at_position, data)
+	elif type == "obj_property":
```

Comment:
Nit: at this point we already know that data has a "type" key, because `_can_drop_data` returned `true`.
```suggestion
	if data["type"] == "nodes":
		_drop_node(at_position, data)
	elif data["type"] == "obj_property":
```

Reply:
Yap!  Fixed.